src/test-suite/src/main.rs

```diff
@@ -744,7 +744,7 @@ mod harness {
 
 #[test]
 fn doc() {
-    expect_test::expect_file!["../README.md"].assert_eq(&cna_util::markdown(
+    expect_test::expect_file!["../API.md"].assert_eq(&cna_util::markdown(
```

Comment:
Moved to files that would more likely be understood as auto-generated (and it's easier to put it to prettify)